doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_external_memory.rst

```diff
@@ -124,10 +124,10 @@ To enable the external memory, you must add the ``-DFILE_SUFFIX="extflash"`` arg
 
    .. code-block:: console
 
-      $ west build -b nrf54h20dk/nrf54h20/cpuapp --sysbuild
+      $ west build -b nrf54h20dk/nrf54h20/cpuapp
       $ west flash
 
-   The build system will automatically generate a :file:`build/suit.zip` archive, which contains the SUIT envelope and candidate images.
+   The build system will automatically use :ref:`configuration_system_overview_sysbuild` and generate a :file:`build/suit.zip` archive, which contains the SUIT envelope and candidate images.
```

Comment:
Please update `build/suit.zip` to `build/dfu_suit.zip`.
I am also currently going through the whole file to verify all others steps, so let's fix only this single issue here :slightly_smiling_face: